documentation/modules/con-iot-concepts.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// assembly-iot-guide.adoc
+
+[id='con-iot-connectivity-{context}']
+= IoT connectivity concepts
+
+IoT connectivity feature enables {ProductName} to be used for managing and connecting devices with backend applications. In a typical IoT application devices are treated differently than ordinary messaging applications. Instead of using arbitrary addresses and security
```

Comment:
```suggestion
The Internet of Things (IoT) connectivity feature enables {ProductName} to be used for managing and connecting devices with back-end applications. In a typical IoT application, devices are configured differently than ordinary messaging applications. Instead of using arbitrary addresses and security
```

Reply:
maybe "have different requirements"

Reply:
@ctron, Good suggestion